src/Form/Form.js

```diff
@@ -584,6 +602,7 @@ class Form {
 
         input.classList.add('ddg-autofilled')
         addInlineStyles(input, getIconStylesAutofilled(input, this))
+        this.touched.add(input)
```

Comment:
Setting all fields to `touched` avoids double prompts on mobile after autofill events.